definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -1,42 +1,42 @@
 [
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "IssueClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DefaultCCJRequested", "UserRole": "caseworker-cmc", "CRUD": "CR"},
```

Comment:
instead of removing these access please  duplicate them for user roles caseworker-cmc-solicitor (upto issue claim),caseworker-cmc-systemupdate,caseworker-cmc-anonymouscitizen. however all these need full testing for  migrated claims and new claims.

Reply:
All documents released event still need to be with caseworker-cmc role

Reply:
If I am duplicating for caseworker-cmc-solicitor (upto issue claim),caseworker-cmc-systemupdate,caseworker-cmc-anonymouscitizen anyway, can I add caseworker-cmc-courtAdmin role as well?

Reply:
yes you can add with all read access